src/app/drive/components/ShareDialog/ShareDialog.tsx

```diff
@@ -133,6 +134,11 @@ const ShareDialog = (props: ShareDialogProps): JSX.Element => {
     return () => clearTimeout(timer);
   }, [accessRequests]);
 
+  useEffect(() => {
+    const currentInvitedUser = invitedUsers.find((user) => user.email === props.user.email);
+    setCurrentUserFolderRole(currentInvitedUser?.roleName);
+  }, [invitedUsers]);
```

Comment:
When is no user invited, at the moment it not appear any user on the list. This causes that the owner couldn't invite users with this logic.

Reply:
Fixed